can-route-pushstate.md

```diff
@@ -8,40 +8,62 @@
 
 @type {RoutePushstate}
 
-__can-route-pushstate__ exports a `RoutePushstate` constructor function that configure [can-route] to use
-[pushstate](https://developer.mozilla.org/en-US/docs/Web/Guide/API/DOM/Manipulating_the_browser_history)
-to change the window's [pathname](https://developer.mozilla.org/en-US/docs/Web/API/URLUtils.pathname) instead
-of the [hash](https://developer.mozilla.org/en-US/docs/Web/API/URLUtils.hash)
+  __can-route-pushstate__ exports a `RoutePushstate` constructor function that configures [can-route] to use
+  [pushState](https://developer.mozilla.org/en-US/docs/Web/Guide/API/DOM/Manipulating_the_browser_history)
+  to change the window's [pathname](https://developer.mozilla.org/en-US/docs/Web/API/URLUtils.pathname) instead
+  of the [hash](https://developer.mozilla.org/en-US/docs/Web/API/URLUtils.hash).
 
-@type {Object} The pushstate object comprises several properties that configure the behavior of [can-route] to work with `history.pushstate`.
+  __can-route-pushstate__ exports an observable that can be used with [can-route]. To start using can-route-pushstate set the [can-route.urlData route.urlData] property:
 
-@body
+  ```html
+  <mock-url pushstate:raw="true"></mock-url>
+  <script type="module">
+  import {route, RoutePushstate} from "can";
+  import "https://unpkg.com/mock-url@next";
 
-## Use
+  route.urlData = new RoutePushstate();
 
-__can-route-pushstate__ exports an observable that can be used with [can-route]. To start using can-route-pushstate set the [can-route.urlData] property:
+  route.register( "{page}", { page: "homepage" } );
+  route.register( "contacts/{username}" );
 
-```js
-import { route, RoutePushstate } from "can";
+  route.start(); // Initializes can-route
 
-route.urlData = new RoutePushstate();
-```
+  route.data.username = "JustinMeyer";
```

Comment:
this doesn't seem to work.  The route is left as `/`

Reply:
Also, the UI seems broken:

![image](https://user-images.githubusercontent.com/78602/48356165-4511b180-e65b-11e8-948d-b7b05d45e74a.png)